public/app/percona/add-instance/components/AddRemoteInstance/AddRemoteInstance.tools.tsx

```diff
@@ -53,3 +88,43 @@ export const getInstanceData = (instanceType: string, credentials: any): Instanc
 
   return instance;
 };
+
+export const getInstanceData = (instanceType: string, credentials: any): InstanceData => {
+  const extractCredentials = (credentials?: any): InstanceData => {
+    if (credentials?.isRDS) {
+      return getRDSCredentials(credentials, instanceType);
+    } else if (credentials?.isAzure) {
+      return getAzureCredentials(credentials, instanceType);
+    }
+
+    const instance: any = { remoteInstanceCredentials: {} };
+
+    switch (instanceType) {
+      case InstanceTypes.postgresql:
+        instance.instanceType = INSTANCE_TYPES_LABELS[InstanceTypes.postgresql];
+        instance.remoteInstanceCredentials.port = 5432;
```

Comment:
Port numbers can go to a constants files with a descriptive name to help understand what they refer to

Reply:
Done